ext/nnstreamer/tensor_decoder/nnstreamer_generated.hh

```diff
@@ -0,0 +1,292 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Don't add generated code in git repo.

Reply:
Updated. 
Add schema file instead of generated file.